### PR TITLE
Publish Latest JSON checklist 2020-11-19

### PR DESCRIPTION
**Pull request by wstgbot**
Updates based on OWASP/wstg@94ce6d9

### CODE REVIEW

File: checklist/checklist.json [hunk 1]
```diff
@@ -695,71 +701,89 @@
...
                 "id":"WSTG-BUSL-09",
                 "reference":"https://owasp.org/www-project-web-security-testing-guide/stable/4-Web_Application_Security_Testing/10-Business_Logic_Testing/09-Test_Upload_of_Malicious_Files.html",
                 "objectives":[
-                    ""
+                    "Identify the file upload functionality.",
+                    "Review the project documentation to identify what file types are considered acceptable, and what types would be considered dangerous or malicious.",
+                    "If documentation is not available then consider what would be appropriate based on the purpose of the application.",
+                    "Determine how the uploaded files are processed.",
+                    "Obtain or create a set of "malicious" files for testing.",
+                    "Try to upload the "malicious" files to the application and determine whether it is accepted and processed."
```
Comment (kingthorin) [inline checklist/checklist.json:786]: Do we need to escape the inner quotes to have syntactically correct JSON?

Comment (kingthorin) [reply checklist/checklist.json:786]: @rejahrehim

Comment (rejahrehim) [reply checklist/checklist.json:786]: Yes. We have to escape. I will send another PR

Comment (ThunderSon) [reply checklist/checklist.json:786]: `jq -R` can do that, so if you want we can fix this for now, and when we update to `jq` this gets fixed. What do you think?

Comment (kingthorin) [reply checklist/checklist.json:786]: I'm merging this one. The job can always be run manually via workflow_dispatch

**Review by kingthorin — APPROVED** [review]

**Pull request merged**